.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    browser: true,
+    commonjs: true,
+    es2021: true,
```

Comment:
what's this for?

Reply:
🤷 must be default setting? deleted anyhow